File: 06_College_Work/Lab_Work/JAVA/Final_CODE/input.java

import java.util.Scanner;

public class input {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String name;
        int age;
        double hight;
        char grade;
        System.out.print("Enter Your name : ");
        name = sc.nextLine();
        System.out.print("Enter your age : ");
        age = sc.nextInt();
        System.out.print("Enter your hight : ");
        hight = sc.nextDouble();
        System.out.print("Enter your grade : ");
        grade = sc.next().charAt(0);
        sc.close();
        
        System.out.println("Hi " + name + " ! Your age is " + age + " Your hight is " + hight+"\nAnd your grade is "+grade);
    }
}

File: 01_Programming_Languages/Dart/Basic/input.dart
import 'dart:io';

void main() {
  print("Enter your name:");
  String? name = stdin.readLineSync();
  print("Enter your age:");

  String? ageInput = stdin.readLineSync();

  int? age = int.tryParse(ageInput ?? '');
  if (age == null) {
    print("Invalid age input.");
    return;
  }
  print("Welcome $name");
}
